blogger-importer.php

```diff
@@ -21,7 +21,12 @@
 // Load Importer API
 require_once ABSPATH . 'wp-admin/includes/import.php';
 
-require_once ABSPATH . WPINC . '/class-feed.php';
+if ( $wp_version >= '4.7.0' ) {

```

Comment:
When comparing two semantic versions, please use the [` version_compare()` function](https://www.php.net/manual/en/function.version-compare.php) as that will be more stable than a comparison operator.

Also, in this case, it might be an idea to replace the version based comparison with a feature based comparison, i.e.:
`if ( file_exists( ABSPATH . WPINC . '/class-simplepie.php' ) ) { ...`
or 
`if ( is_readable( ABSPATH . WPINC . '/class-simplepie.php' ) ) { ...`

Reply:
Hi. 
The class-simplepie.php existed in versions <4.7. 
Should I make a commit with version_compare() function ?